engine.py

```diff
@@ -504,9 +504,53 @@ def save_to_path(self, document, path):
 
             # make the document being processed the active document
             self.adobe.app.activeDocument = document
-
-            # TODO need to use appropriate save options here (jpg, tiff, etc)
-            document.saveAs(self.adobe.File(path))
+        
+            (_, ext) = os.path.splitext(path)
+
+            # valid file extensions and respective save option in photoshop 
+            valid_extensions = {
+                                 ".bmp" : self.adobe.BMPSaveOptions, 
+                                 # DCS1_saveOptions and DCS2_SaveOptions skipped
```

Comment:
maybe move all the "skipped" comments to the comment above the declaration with an explanation as to why we're skipping some.